tests/integration/telemetry/stackdriver/stackdriver_filter_test.go

```diff
@@ -253,3 +212,153 @@ func testSetup(ctx resource.Context) (err error) {
 	}
 	return nil
 }
+
+// send both a grpc and http requests (http with forced tracing).
+func sendTraffic(t *testing.T) error {
+	t.Helper()
+	grpcOpts := echo.CallOptions{
+		Target:   srv,
+		PortName: "grpc",
+		Count:    1,
+	}
+	if _, err := clt.Call(grpcOpts); err != nil {
+		return err
+	}
+
+	// an HTTP request with forced tracing
+	hdr := http.Header{}
+	hdr.Add("x-envoy-force-trace", "true")
```

Comment:
with pilot sampling set to 100, this should not be needed?

Reply:
yes, though I think our framework generates its own `x-request-id`s that encode sampling as off as a side-effect. I think i can remove the pilot sampling config and just go with the force-trace stuff.

Reply:
Removed the pilot sampling bits for now.